integration/spark2/pom.xml

```diff
@@ -376,6 +385,55 @@
                   <sources>
                     <source>src/main/spark2.3</source>
                     <source>src/main/commonTo2.2And2.3</source>
+                    <source>src/main/commonTo2.2AndAbove</source>
```

Comment:
It becomes so messy to add so many folders, when we are planning to remove 2.1 and 2.2 support?

Reply:
In 2.0 we will remove spark 2.1 and spark 2.2 support. I am not sure whether we should delete the code or just keep it regardless whether it is not compilable

Reply:
Removed spark 2.1 and 2.2 integration support in this PR